src/Examine.Lucene/Suggest/LuceneSuggestionResults.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Examine.Suggest;
+
+namespace Examine.Lucene.Suggest
+{
+    public class LuceneSuggestionResults : ISuggestionResults
```

Comment:
This is public and should therefore have XML documentation - Use `/// <inheritdoc />` where appropriate

Reply:
Done